src/django_upgrade/data.py

```diff
@@ -38,6 +38,10 @@ def __init__(
     def looks_like_migrations_file(self) -> bool:
         return "migrations" in self.filename.split(os.path.sep)
 
+    def looks_like_admin_command(self) -> bool:
+        path_parts = self.filename.split(os.path.sep)
+        return "management" in path_parts and "commands" in path_parts
```

Comment:
I think to be even surer, we could check for "management" followed by "commands" - perhaps with clever use of `list.index()`